pkg/validate/validate.go

```diff
@@ -184,9 +184,39 @@ func IP(ip string) error {
 	return nil
 }
 
-// MAC validates that a value is a valid mac address
+// MAC validates that a value is a valid unicast EUI-48 MAC address
 func MAC(addr string) error {
-	_, err := net.ParseMAC(addr)
+	hwAddr, err := net.ParseMAC(addr)
```

Comment:
Rather than checking `err == nil` repeatedly, return early if `err != nil`.
```suggestion
	hwAddr, err := net.ParseMAC(addr)
	if err != nil {
		return err
	}
```

Reply:
Makes sense, thanks